### ISSUE TITLE
Profiles for SEO agencies, consultants, freelancers

### ISSUE THREAD

**Issue by jsilton** [SEO-Community/seo-community#6]
Should we all the community to create profiles for SEO agencies, consultants, and freelancers? Maybe each company could also be free to edit their page too?

**Comment by zldoty** [comment]
@jsilton Definitely! Knowing what we know about the ambition to boldly optimize within our industry, shall we set some governance / structure around profile creation? Ex: a panel approval of new profiles, some reasonable constraints about linking back to their site?

**Comment by jsilton** [comment]
As a starting point, I think we could it open since it should, in theory, self-moderate with entries people are willing to vouch for. Any contribution can be attributed back to the code commit.

If people abuse this, I think the community will propose some changes, or just make changes. I would suggest more Wikipedia style than moderators. Does that make any sense?

**Comment by zldoty** [comment]
Sounds good to me! Good point on the commit being tied back to someone. :-)
